.github/workflows/ContinuousIntegration.yml

```diff
@@ -0,0 +1,30 @@
+# Adapted from
+# https://github.com/marketplace/actions/run-julia-package-tests
+
+name: ContinuousIntegration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ContinuousIntegration:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.3', 'nightly']
```

Comment:
`1.3` is quite old -- most packages in the ecosystem have dropped support for `1.3`. Let's test the latest LTS instead `1.6.7`.

Reply:
I am happy to bump it, but also not sure why the current Travis file tests against 1.3, maybe it is related to how Gen is configured in the official Julia Registry?

https://github.com/JuliaRegistries/General/blob/01717d7c358686ddbbe8ce40d159b6452b870f52/G/Gen/Compat.toml#L33-L35

Reply:
It's not about the Julia Registry, it's that the current version of Gen (v0.4) still officially supports Julia 1.3 according to our `Project.toml` file: https://github.com/probcomp/Gen.jl/blob/master/Project.toml

We could decide to drop support for Julia 1.3 and only support Julia 1.6 onwards, but I think that should be a separate change, and in the meantime I think our CI tests should still test on the versions we're claiming to support!

Reply:
Agreed, let's keep 1.3 for now and handle the upgrade to 1.6 in a future PR.

Reply:
So it looks for the Travis file, we are currently testing 1.3, 1.6, 1 (latest stable), and nightly -- I think we should do the same for the GitHub CI action.

Reply:
Testing against fewer versions has the advantage that it is faster to get the status checks (where each version can take several minutes) and saves computational resources, especially when the status checks are run against each commit in a PR (which is something we should disable).

For this PR I'm happy to test against the same versions of Travis, and then we can do a separate PR for deciding which versions CI should test against and formally support (e.g., "1.6" and "nightly").